packages/terra-notification-dialog/src/NotificationDialog.jsx

```diff
@@ -223,6 +223,14 @@ class NotificationDialog extends React.Component {
       message,
       ...customProps
     } = this.props;
+    if (acceptAction === undefined && primaryAction === undefined) {
+      // eslint-disable-next-line no-console
+      console.warn('acceptAction or primaryAction prop must be given.');
```

Comment:
We don't want to log these messages in production mode. Lets add:
```
if (process.env.NODE_ENV !== 'production') {
     console.warn(....)
}
```

Reply:
Also can we add some context to the message. A consumer that didn't set these props may not know these messages are meant for notification-dialog

Reply:
Changed in 2957b32